charts/coturn/templates/statefulset.yaml

```
@@ -116,6 +121,11 @@ spec:
               mountPath: /secrets-tls/
               readOnly: true
           {{- end }}
+{{- if .Values.federate.dtls.enabled }}
+            - name: coturn-dtls-certificate
+              mountPath: /coturn-dtls-certificate/
+              readOnly: true
+{{- end }}
```

consistent indentation?

```suggestion
           {{- if .Values.federate.dtls.enabled }}
            - name: coturn-dtls-certificate
              mountPath: /coturn-dtls-certificate/
              readOnly: true
           {{- end }}
```